README.md

```diff
@@ -114,6 +116,46 @@ To stop the containers run:
 
 ## Application Concepts
 
+## Usage
+
+This API was built following REST pattern, then you can access all cruds this way:
+
+- Companies:
+
+  index: GET http://localhost:3000/companies
+  show: GET http://localhost:3000/companies/id
+  showcase: GET http://localhost:3000/showcase
+  create: POST http://localhost:3000/companies
```

Comment:
It would be really nice if we have some payload examples for the POST actions 😄
For example:
```json
{
	"user": {
		"email": "manager@example.com",
		"password": "123456"
	}
}
```